portality/templates/500.html

```diff
@@ -0,0 +1,10 @@
+{% extends "/base.html" %}
+
+{% block content %}
+
+<h2>Oops. Something went wrong.</h2>
+
+<p>An error has occurred. It looks like something has gone wrong.</p>
+<p>Please contact us using the Contact link at the bottom od the page.</p>
```

Comment:
Typo, and we could improve these templates by including the actual contact us URL (I'm not sure why we currently direct the user to click the other link, it's probably bad design or something)